config/sync/field.field.node.article.field_show_overwrite_author.yml

```diff
@@ -0,0 +1,23 @@
+uuid: eafb283f-7922-45bd-a3ca-692b69c2160a
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_show_overwrite_author
+    - node.type.article
+id: node.article.field_show_overwrite_author
+field_name: field_show_overwrite_author
+entity_type: node
+bundle: article
+label: 'Overwrite the author'
+description: "By default, the author is set to the Drupal user that owns the content.<br>\r\nIf you want to override this, with a manual text, you can check this."
```

Comment:
Please update this field to use [override instead of overwrite](https://stackoverflow.com/questions/8651562/overwrite-or-override).

As I see it this description gets it right, the label (and field id) gets it wrong.